scripts/gitlab/build_and_test.sh

```diff
@@ -206,21 +233,32 @@ then
 
     if grep -q "Errors while running CTest" ./tests_output.txt
     then
-        echo "ERROR: failure(s) while running CTest" && exit 1
+        echo "[Error]: failure(s) while running CTest" && exit 1
     fi
 
-    if [[ ! -d ${install_dir} ]]
+    if grep -q -i "ENABLE_HIP.*ON" ${hostconfig_path}
```

Comment:
I brought this over from Umpire and RAJA. Not sure if it's actually necessary.

Reply:
Hmm, if this wasn’t an issue so far, I wouldn’t add a new restriction here. Instead, I would ask @rhornung67 and @davidbeckingsale if we still can’t test the install when using HIP, and why.

Reply:
I reverted this change and if the pipelines pass, we'll know it's not needed in CHAI.